pkg/features/list.go

```diff
@@ -54,7 +54,7 @@ var (
 	ProcessesListeningOnPort = registerFeature("Enable Processes Listening on Port", "ROX_PROCESSES_LISTENING_ON_PORT", false)
 
 	// ClairV4Scanner enables Clair v4 as an Image Integration option
-	ClairV4Scanner = registerFeature("Enable Clair v4 as an Image Integration option", "ROX_CLAIR_V4_SCANNING", false)
+	ClairV4Scanner = registerFeature("Enable Clair v4 as an Image Integration option", "ROX_CLAIR_V4_SCANNING", true)
```

Comment:
is there a way to feature flag the qa groovy test?

Reply:
I think https://github.com/stackrox/stackrox/blob/f8bdc4ea695505af176c85cd15de7699f0f677e0/tests/e2e/lib.sh#L85 if you want it for all e2e.